packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -313,6 +321,9 @@ export default {
                 if (!this.$v.email.email) {
                     return messages.invalidEmailError;
                 }
+                if (!this.$v.email.emailNotConflicted) {
```

Comment:
This is a bit hard to read. "Not email not conflicted".
Could we do something like "isEmailValid" instead?